napari/_commands/_tests/test_commands.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+from napari._commands import CommandProcessor
+
+
+def test_help(make_test_viewer, capsys):
+    """Test cmd.help."""
+    viewer = make_test_viewer()
+    viewer.cmd.help
+    out, _ = capsys.readouterr()
+    assert out.count('\n') >= 4
+
+
+def test_no_layers(make_test_viewer, capsys):
+    """Test cmd.layer with no layers."""
+    viewer = make_test_viewer()
+    CommandProcessor(viewer.layers).layers
+    out, _ = capsys.readouterr()
+    for x in ["ID", "NAME", "LAYER"]:  # Just check a few.
+        assert x in out
+
+
+def test_one_layer(make_test_viewer, capsys):
+    """Test cmd.layer with one layer."""
+    viewer = make_test_viewer()
+    data = np.random.random((10, 15))
+    viewer.add_image(data, name="pizza")
+    viewer.cmd.layers
+    out, _ = capsys.readouterr()
+    assert "pizza" in out
+    assert "(10, 15)" in out
+
```

Comment:
We might also want to add a test that cycles through all our standard `layer_data` examples, see here https://github.com/napari/napari/blob/master/napari/_tests/utils.py#L8-L28 and https://github.com/napari/napari/blob/c2c6b8e32f79397225c8883ce03a8900b8708a94/napari/layers/_tests/test_serialize.py#L4-L13 for usage. This would catch the error with the surface layer that happens now (see comments above).